src/index.html

```diff
@@ -15,18 +15,38 @@
       rel="stylesheet"
       href="style.css"
     />
+    <link
+      rel="stylesheet"
+      href="fonts/Avenir-Book.ttf"
+    />
```

Comment:
The `href` attribute for fonts should point to a stylesheet (CSS file) that includes the font-face declaration, not the font file itself. Make sure you link to a `.css` file that defines the font-face.